content/terms/contributor-guide/_index.md

```diff
@@ -6,31 +6,25 @@ Welcome to Tinkerbell! We're building the industry standard for bare metal infra
 
 ## Tinkerbell Projects
 
-Tinkerbell is a bare metal provisioning engine and consists of five components: a DHCP server (Boots), a metadata service (Hegel), an in-memory operating system installation environment (OSIE), a workflow engine (Tink), and a Power and Boot service (PBnJ) that communicates with BMCs. Learn more about these and other repos we host:
+Tinkerbell is a bare metal provisioning engine and consists of various components, some of which have pluggable alternatives.
 
-|                          Project Name                          |                                  Description                                  |
```

Comment:
I think this table still offers some value, this is the guide for folks that want to contribute but aren't sure which repo is responsible for what.

Perhaps the in-paragraph list of components could be replaced with your pluggable alternatives statement, the table could be revised with the most common projects and their descriptions, and the org link could be provided after the table?

Reply:
I like that, will update.